### PR TITLE
[6]Develop a Product Page Using HTML

### PR DESCRIPTION
**Pull request by Fiyin985**
## Task: `Making product page Using HTML`
-Changed the name of id
Screenshot (14)
Screenshot (13)

### CODE REVIEW

File: submissions/fiyingabriel/HTML/prd for html task.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE HTML>
```
Comment (ToluAkin) [inline submissions/fiyingabriel/HTML/prd for html task.html:1]: You need to remove all the styles on this file

File: submissions/fiyingabriel/HTML/prd for html task.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	<meta charset="utf-8">
+</head>
+  
+<body>
+	<p> <img src="tte.jpg" height="200" width="2500"> <img src="OIP.jpg" height="100"width="300"></p>
+	<h1>Apple iPhone X - 64GB</h1>
+	<div>
+    <p><img src="149543_1549727850.jpg" height="500px" width="500px"></p>
+    <h1 align="left">N250,000</h1>
+    <img src ="BuyNowButton.png"height = "100" width="200">
```
Comment (ToluAkin) [inline submissions/fiyingabriel/HTML/prd for html task.html:16]: You are meant to use only 2 product images. Check the criteria here.
https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp20/issues/7
You should the images that you are not using.

Comment (Fiyin985) [reply submissions/fiyingabriel/HTML/prd for html task.html:16]: I only used 2 product images

Comment (Fiyin985) [reply submissions/fiyingabriel/HTML/prd for html task.html:16]: I dont understand

Comment (Fiyin985) [reply submissions/fiyingabriel/HTML/prd for html task.html:16]: this is the other half of the page
Screenshot (14)

Comment (ToluAkin) [reply submissions/fiyingabriel/HTML/prd for html task.html:16]: You should delete the images you're not using that you've added .

Comment (Fiyin985) [reply submissions/fiyingabriel/HTML/prd for html task.html:16]: Im using all of them

Comment (Fiyin985) [reply submissions/fiyingabriel/HTML/prd for html task.html:16]: Which one should i remove

**Pull request closed without merging**